src/components/shared/Tray/TrayComponents/SelectedService/InfoAboutSelectedService/TramInfoAboutSelectedService/TramInfoAboutSelectedService.js

```diff
@@ -0,0 +1,79 @@
+import React, { useContext } from 'react';
+import { AutoCompleteContext } from 'globalState';
+// Components
+import FavBtn from 'components/shared/FavBtn/FavBtn';
+import Message from 'components/shared/Message/Message';
+// Local
+import numberToWord from '../helpers/numberToWord';
+import useGetTramStopByStop from './customHooks/useGetTramStopByStop';
+
+const TramInfoAboutSelectedService = () => {
+  const [autoCompleteState] = useContext(AutoCompleteContext);
+  const { selectedItem, selectedItemTo } = autoCompleteState;
+  const { loading, errorInfo } = useGetTramStopByStop();
```

Comment:
To tidy this file a bit more, return the autoCompleteState from the useGetTramStopByStop.